bool autopilot_connected
bool receiving_position
bool receiving_attitude
# TOBE extended
